packages/@vue/cli-service-global/template/index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="">
```

Comment:
Being an empty attribute should be enough indication for developers to insert the correct language themselves.